odl/tomo/backends/astra_setup.py

```diff
@@ -117,14 +181,16 @@ def astra_volume_geometry(discr_reco):
 
     if discr_reco.ndim == 2:
         # ASTRA does in principle support custom minimum and maximum
-        # values for the volume extent, but projector creation fails
+        # values for the volume extent, but running the algorithm fails
         # if voxels are non-isotropic. We raise an exception here in
         # the meanwhile.
-        if not np.allclose(discr_reco.partition.cell_sides[1:],
-                           discr_reco.partition.cell_sides[:-1]):
-            raise NotImplementedError('non-isotropic voxels not supported by '
-                                      'ASTRA')
-        # given a 2D array of shape (x, y), a volume geometry is created as:
+        if (not np.allclose(discr_reco.partition.cell_sides[1:],
```

Comment:
We should add something like "is_isotropic_cell_sides" to partition instead of this.